lib/open_food_network/order_and_distributor_report.rb

```diff
@@ -28,6 +28,7 @@ def header
         I18n.t(:report_header_distributor_address),
         I18n.t(:report_header_distributor_city),
         I18n.t(:report_header_distributor_postcode),
+        I18n.t(:report_header_shipping_method),
```

Comment:
Interesting, this line has failed with rubocop. I believe it wants everything aligned with the first line of the array.

Let me know if this is right, and I will amend!

Reply:
the problem was already there, you can fix it by making this header method similar to the row_for method
```
def header
  [
    I18n.t(...),
    I18n.t(...)
  ]

```
if you do this you can also delete this line from the exceptions:
https://github.com/openfoodfoundation/openfoodnetwork/blob/fed2b6074320f814daac822fd6c2e0020a1749bd/.rubocop_todo.yml#L23

Reply:
Easy enough! Fixing this now.